data_structures/linked-list/linked_list.py

```diff
@@ -0,0 +1,41 @@
+from node import Node
+
+
+class LinkedList:
+    """Class for a singly linked list."""
+
+    def __init__(self, iter=[]):
+        """Initialize the list."""
+        self._current = None
+        self.head = None
+        self._size = 0
+        try:
+            for item in iter:
+                self.insert(item)
+        except ValueError:
+            print('Value error! Please insert an iterable.')
+
+    def __repr__(self):
+        """Return the value of the head."""
+        try:
+            return '<head> => {}'.format(self.head.val)
+        except ValueError:
+            print('Oops! The head has no value.')
```

Comment:
Make sure `__repr__` returns a string at all times. You might want to show that the `head` is currently `None` or something of the sort instead of printing something.